src/scenes/Languages/LanguageForm/LanguageForm.tsx

```diff
@@ -48,212 +68,267 @@ export const LanguageForm = <T extends any>({
-                        label="Public Name"
-                        placeholder="Enter Public Name"
-                        required
-                        validate={[required, minLength()]}
-                        margin="none"
-                        {...props}
-                      />
-                    </Grid>
-                  )}
-                </SecuredField>
-                <SecuredField obj={language} name="displayNamePronunciation">
-                  {(props) => (
-                    <Grid item xs={12}>
-                      <TextField
-                        label="Pronunciation Guide"
-                        placeholder="Enter Pronunciation of Public Name"
-                        margin="none"
-                        {...props}
-                      />
-                    </Grid>
-                  )}
-                </SecuredField>
-                <SecuredField obj={language} name="registryOfDialectsCode">
-                  {(props) => (
-                    <Grid item xs={12} sm={6}>
-                      <FormattedTextField
-                        label="Registry Of Dialects Code"
-                        placeholder="#####"
-                        accept={/\d/g}
-                        formatInput={(value) =>
-                          (value.match(/[\d]+/g) || []).join('').substr(0, 5)
-                        }
-                        validate={(value) =>
-                          !value || value.length === 5
-                            ? undefined
-                            : `Must be 5 digits`
-                        }
-                        margin="none"
-                        {...props}
-                      />
-                    </Grid>
-                  )}
-                </SecuredField>
-                <SecuredField obj={language} name="populationOverride">
-                  {(props) => (
-                    <Grid item xs={12} sm={6}>
-                      <NumberField
-                        label="Population"
-                        placeholder="Enter Population"
-                        helperText="Leave blank to use population from Ethnologue"
-                        margin="none"
-                        {...props}
-                      />
-                    </Grid>
-                  )}
-                </SecuredField>
-                <SecuredField obj={language} name="isDialect">
-                  {(props) => (
-                    <Grid item xs={12} sm={6}>
-                      <CheckboxField
-                        label="Is this language a dialect?"
-                        margin="none"
-                        {...props}
-                      />
-                    </Grid>
-                  )}
-                </SecuredField>
-              </Grid>
+      {({
+        values,
+      }: {
+        values: Partial<LanguageFormValues<CreateLanguage | UpdateLanguage>>;
+      }) => {
+        console.log('values: ', values);
```

Comment:
```suggestion
```

Reply:
oops fixed.  About time we put in that eslint console log warning?

Reply:
Maybe! It can't annoy me too much more than the one for `debugger`